machine/v1/types_controlplanemachineset.go

```diff
@@ -343,12 +343,15 @@ type RootVolume struct {
 	AvailabilityZone string `json:"availabilityZone,omitempty"`
 
 	// volumeType specifies the type of the root volume that will be provisioned.
-	// If not specifified, the root volume will be created as the type in the machine template.
+	// Historically, the installer has always required a volume type to be specified when deploying
+	// the control plane with a root volume. This is because the default volume type in Cinder is not guaranteed
+	// to be available, therefore we prefer the user to be explicit about the volume type to use.
+	// We apply the same logic in CPMS: if the failure domain specifies a root volume, we require the user to specify a volume type.
 	// The maximum length of a volume type name is 255 characters, as per the OpenStack limit.
+	// +kubebuilder:validation:Required
```

Comment:
`RootVolume` still has `+kubebuilder:validation:MinProperties:=1`, which I'm not sure it makes sense anymore but I suspect this doesn't hurt.

Reply:
Yeah I think lets drop the min properties since it has no effect now